### DISCUSSION TITLE
Multiple ArgoCDs (clusters) Single View

### DISCUSSION THREAD

**Discussion by zaphinath** [argoproj/argo-cd#15215] [General]
Due to various security issues and lazy development we ended up with multiple clusters and some have their own argocd and some share an argocd. Is there a way today to unify all of them into a single view? Searching the internet and chatgpt have not yielded any fruit on this question. Thanks.

**Comment by FernandoMiguel** [comment]
I would say the target is to have a single Argo CD, so that is your single view.
a mix env of single and individual, seems complex
